### PR TITLE
add_working_with_data_docs

### PR DESCRIPTION
**Pull request by claudiaregio**

### CODE REVIEW

File: docs/Using Notebooks.md/Working with Data.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+## Querying
+
+To connect to data and query in Polyglot Notebooks in VS Code, follow the steps below: 
+
+1. Create a new Polyglot Notebook starting with a C# cell. 
+
+1. In the first cell, use the required Nuget package from above. Make sure this has it's own cell and run it. 
+
+1. Add another C# cell and use the corresponding `#!connect` command from above to establish your connection and run the cell.
+    - The `--kernel-name` parameter is used to alias your data connection. This is the name you'd like to refer to this connection as while you continue to work in your notebook. It will become available in the dynamic kernel picker in the bottom right of cells.
```
Comment (AbhitejJohn) [inline docs/Using Notebooks.md/Working with Data.md:35]: nit nit: The `as` can be dropped - "refer to this connection ~~as~~ while you continue"

File: docs/Using Notebooks.md/Working with Data.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+![image](https://user-images.githubusercontent.com/19276747/207750707-c227d359-1a25-4cc7-875d-b2dc056ccbe6.png)
+
+Kusto Example
+
+![image](https://user-images.githubusercontent.com/19276747/207726856-343eff43-2f93-49d7-a747-21c4ccc80033.png)
+
+
+# Storing and Sharing Query Results
+
+Since Polyglot Notebooks not only allows you to use different languages in the same notebook but share variables between them, you might be interested in storing query results to pass of beteween language to language.
```
Comment (AbhitejJohn) [inline docs/Using Notebooks.md/Working with Data.md:54]: nit nit nit: `beteween` -> `between`

File: docs/Using Notebooks.md/Working with Data.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+The syntax to store MSSQL or KQL queries: 
+
+>`#!connection-alias --name` name
+
+The syntax to share variables from a kernel connection into a new kernel:  
+
+>`#!share --from connection-alias` name
+
+*Note: The kernel the variable is being shared **to** is determined by the selection of the dynamic kernel picker in the bottom right of each cell.
```
Comment (AbhitejJohn) [inline docs/Using Notebooks.md/Working with Data.md:64]: nit: Did you mean to bold the Note?

File: docs/Using Notebooks.md/Working with Data.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+1. Add another C# cell and use the corresponding `#!connect` command from above to establish your connection and run the cell.
+    - The `--kernel-name` parameter is used to alias your data connection. This is the name you'd like to refer to this connection as while you continue to work in your notebook. It will become available in the dynamic kernel picker in the bottom right of cells. 
+    - If this is your first time establishing this connection, a browser window will open for authentication.    
+ 
+ 1. Your connection to a Microsoft SQL Server Database or Kusto Cluster Database is now complete! To query this database, simply add a new cell and click the dynamic kernel picker in the bottom right. Select the connection alias you created, and write your raw SQL or KQL code. 
+
+
+## Examples
+
+MS SQL Example
```
Comment (AbhitejJohn) [inline docs/Using Notebooks.md/Working with Data.md:43]: consider: For someone to re-use these examples, could we also put in the code for the statements here?

**Review by AbhitejJohn — APPROVED** [review]

File: docs/Using Notebooks.md/Working with Data.md (hunk 1 above)
Comment (claudiaregio) [reply docs/Using Notebooks.md/Working with Data.md:35]: I think it is needed...reads weird to me without it

File: docs/Using Notebooks.md/Working with Data.md (hunk 4 above)
Comment (claudiaregio) [reply docs/Using Notebooks.md/Working with Data.md:43]: I can upload the mini notebooks in our samples and link them here when i'm back

File: docs/Using Notebooks.md/Working with Data.md (hunk 2 above)
Comment (claudiaregio) [reply docs/Using Notebooks.md/Working with Data.md]: fixed

File: docs/Using Notebooks.md/Working with Data.md (hunk 3 above)
Comment (claudiaregio) [reply docs/Using Notebooks.md/Working with Data.md:64]: No, just put an asterisk next to it. Do you think it needs to be bolded?

**Pull request merged**